modules/aws-limits.adoc

```diff
@@ -18,13 +18,14 @@ The following table summarizes the AWS components whose limits can impact your a
 |Instance Limits
 |Varies
 |Varies
-|By default, each cluster creates the following instances:
+|At a minimum, each cluster creates the following instances:
 
 * One bootstrap machine, which is removed after installation
 * Three master nodes
-* Three worker nodes
+* Three infrastructure nodes
```

Comment:
Can we make this **Two** infrastructure nodes?  2 is the min for a single az and 3 for multi.

Reply:
@0kashi, I changed it to two but do you want me to write that it's two for single and three for multi?

Reply:
I think that's a good idea.

Reply:
Excellent! I updated the PR to include that information. Can you check to make sure it makes sense in the way I presented it?